internal/appconfig/config.go

```diff
@@ -25,6 +25,14 @@ const (
 	DetachedPlatform = "detached"
 )
 
+type RestartPolicy string
+
+const (
+	RestartPolicyAlways    RestartPolicy = "always"
+	RestartPolicyNever     RestartPolicy = "no"
```

Comment:
Can we make users write `policy = "never"` instead of `policy = "no"`?

Reply:
I want to change it everywhere so it doesn't bite us in the future.

Reply:
Sorry, not sure what this means

Reply:
I was thinking something like
```patch
diff --git a/internal/appconfig/config.go b/internal/appconfig/config.go
index 2e221321..4ff5d650 100644
--- a/internal/appconfig/config.go
+++ b/internal/appconfig/config.go
@@ -29,7 +29,7 @@ type RestartPolicy string
 
 const (
 	RestartPolicyAlways    RestartPolicy = "always"
-	RestartPolicyNever     RestartPolicy = "no"
+	RestartPolicyNever     RestartPolicy = "never"
 	RestartPolicyOnFailure RestartPolicy = "on-failure"
 )
 
diff --git a/internal/appconfig/machines.go b/internal/appconfig/machines.go
index c0f5a9c8..c6135562 100644
--- a/internal/appconfig/machines.go
+++ b/internal/appconfig/machines.go
@@ -1,6 +1,7 @@
 package appconfig
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/google/shlex"
@@ -194,14 +195,31 @@ func (c *Config) updateMachineConfig(src *api.MachineConfig) (*api.MachineConfig
 	mConfig.Restart = nil
 
 	if c.Restart != nil {
+		policy, err := parseRestartPolicy(c.Restart.Policy)
+		if err != nil {
+			return nil, err
+		}
 		mConfig.Restart = &api.MachineRestart{
-			Policy:     api.MachineRestartPolicy(c.Restart.Policy),
+			Policy:     policy,
 			MaxRetries: c.Restart.MaxRetries,
 		}
 	}
 	return mConfig, nil
 }
 
+func parseRestartPolicy(policy RestartPolicy) (api.MachineRestartPolicy, error) {
+	switch policy {
+	case RestartPolicyAlways:
+		return api.MachineRestartPolicyAlways, nil
+	case RestartPolicyOnFailure:
+		return api.MachineRestartPolicyOnFailure, nil
+	case RestartPolicyNever:
+		return api.MachineRestartPolicyNo, nil
+	default:
+		return "", errors.New("invalid restart policy")
+	}
+}
+
 func (c *Config) tomachineSetStopConfig(mConfig *api.MachineConfig) error {
 	mConfig.StopConfig = nil
 	if c.KillSignal == nil && c.KillTimeout == nil {
```